content/authors/daniela-brozzoni/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Daniela Brozzoni
 photo:
-twitter: danielabrozzoni
+twitter: daniela-brozzoni
```

Comment:
My handle is still `danielabrozzoni`, can you remove this? :)

Reply:
ohh shoot, fixed.